packages/zowe-explorer-api/src/utils/index.ts

```diff
@@ -9,5 +9,18 @@
  *
  */
 
+import { IZoweTreeNode } from "../tree";
+import { workspace } from "vscode";
+
 export * from "./Poller";
 export * from "./FileManagement";
+
+/**
+ * Getter to check dirty flag for nodes opened in the editor.
+ *
+ * NOTE: Only works for nodes that use resource URIs (see the `resourceUri` variable in IZoweTreeNode)
+ * @returns {boolean} whether the URI is open in the editor and unsaved
+ */
+export function isNodeInEditor(node: IZoweTreeNode): boolean {
```

Comment:
This is also not in the changelog 😅 
If we don't plan to add it as a small enhancement in the changelog, should we make it `internal`